Automattic/wp-calypso
# WP for Teams: add initial signup flow

In this PR, we add the initial version of signup flow for the [WP for Teams project](https://wp.me/p9lV3a-1dM-p2).

## Testing instructions

Navigate to `http://calypso.localhost:3000/start/wp-for-teams`. You should be welcomed with the "Choose a site address" input which by default uses the wpcom Free plan. The `pub/p2020` theme should be pre-selected and activated once you finish the flow.

All the other flows, especially the ones using the `site` step should work as before.
